cpp/cmake_modules/FindQPL.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright 2012 Cloudera Inc.
```

Comment:
Where did you copy from?

Reply:
I copied from findThrift.cmake.  I'll remove this line.